spring-data-relational/src/main/java/org/springframework/data/relational/repository/query/RelationalExampleMapper.java

```diff
@@ -64,92 +72,194 @@ public <T> Query getMappedExample(Example<T> example) {
 	 * {@link Query}.
 	 *
 	 * @param example
-	 * @param entity
+	 * @param persistentEntity
 	 * @return query
 	 */
-	private <T> Query getMappedExample(Example<T> example, RelationalPersistentEntity<?> entity) {
+	private <T> Query getMappedExample(Example<T> example, RelationalPersistentEntity<?> persistentEntity) {
```

Comment:
What's the reason behind the renaming?

Reply:
Because one can confuse the `PersistenceEntity` with the `entity` as the actual instance/probe. I do not insist here at all, just my observation when working with code, what do you think, @mp911de ?